### PR TITLE
feat: German Translations Update By Andreas Plank

### PR DESCRIPTION
**Pull request by Tirokk**
> [human-connection-release-bot](https://github.com/human-connection-release-bot) **Authored by [human-connection-release-bot](https://github.com/human-connection-release-bot)**
_<time datetime="2020-02-19T09:27:53Z" title="Wednesday, February 19th 2020, 10:27:53 am +01:00">Feb 19, 2020</time>_
_Merged <time datetime="2020-02-19T22:33:57Z" title="Wednesday, February 19th 2020, 11:33:57 pm +01:00">Feb 19, 2020</time>_
---

This pull request was initiated by Lokalise (user Mogge) at 2020-02-19 10:27:49

### CODE REVIEW

**Comment by Tirokk** [comment]
> [cypress[bot]](https://github.com/apps/cypress) **Authored by [cypress[bot]](https://github.com/apps/cypress)**
_<time datetime="2020-02-19T12:58:23Z" title="Wednesday, February 19th 2020, 1:58:23 pm +01:00">Feb 19, 2020</time>_

---



<p align="center">
  <br />
  
</p>

---

### Test summary
<a href="https://dashboard.cypress.io/projects/qa7fe2/runs/5355/specs"></a> <b title="Passed">66</b>  &bull; <a href="https://dashboard.cypress.io/projects/qa7fe2/runs/5355/failures"></a> <b title="Failed">0</b>  &bull;  <b title="Pending">0</b> &bull;  <b title="Skipped">0</b>

---

### Run details
| | |
| --- | --- |
| **Project** | Human-Connection |
| **Status** | Passed |
| **Commit** | 87ad57851c |
| **Started** | Feb 19, 2020 7:30 PM |
| **Ended** | Feb 19, 2020 7:44 PM |
| **Duration** | 13:46 <a href="https://on.cypress.io/parallelization" title="Enable parallelization to reduce run duration">💡</a> |
| **OS** | Linux Ubuntu Linux - 16.04 |
| **Browser** | Electron 78  |

[View run in Cypress Dashboard ➡️](https://dashboard.cypress.io/projects/qa7fe2/runs/5355/specs)



---

This comment has been generated by **cypress-bot** as a result of this project's GitHub integration settings. You can manage this integration in [this project's settings in the Cypress Dashboard](https://dashboard.cypress.io/projects/qa7fe2/settings)

File: webapp/locales/de.json [hunk 1]
```diff
@@ -40,7 +40,7 @@
       "tagCountUnique": "Benutzer"
     },
     "invites": {
-      "description": "Einladungen sind ein wunderbarer Weg, deine Freund in deinem Netzwerk zu haben …",
+      "description": "Einladungen sind eine wunderbare Möglichkeit, Deine Freunde in Deinem Netzwerk zu haben …",
```
Comment (Tirokk) [inline webapp/locales/de.json:43]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-19T22:29:06Z" title="Wednesday, February 19th 2020, 11:29:06 pm +01:00">Feb 19, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -40,7 +40,7 @@
       "tagCountUnique": "Benutzer"
     },
     "invites": {
-      "description": "Einladungen sind ein wunderbarer Weg, deine Freund in deinem Netzwerk zu haben …",
+      "description": "Einladungen sind eine wunderbare Möglichkeit, Deine Freunde in Deinem Netzwerk zu haben …",
```


It's OK for me now to use the capitalized `Du` here but according to a teacher of journalism, this spelling is antiquated and except for letters it's now common to use lowercase `du`. So for the future, I would prefer that.

File: webapp/locales/de.json [hunk 2]
```diff
@@ -635,8 +635,8 @@
       "success": "Deine Daten wurden erfolgreich aktualisiert!"
     },
     "deleteUserAccount": {
-      "accountDescription": "Sei dir bewusst, dass deine Beiträge und Kommentare für unsere Community wichtig sind. Wenn du sie trotzdem löschen möchtest, musst du sie unten markieren.",
-      "accountWarning": "Dein Konto, deine Beiträge oder Kommentare kannst du nach dem Löschen <b>WEDER VERWALTEN NOCH WIEDERHERSTELLEN!</b>",
+      "accountDescription": "Sei Dir bewusst, dass Deine Beiträge und Kommentare für unsere Community wichtig sind. Wenn Du sie trotzdem löschen möchtest, musst Du sie unten markieren.",
+      "accountWarning": "Dein Konto, Deine Beiträge oder Kommentare kannst Du nach dem Löschen <b>WEDER VERWALTEN NOCH WIEDERHERSTELLEN!</b>",
```
Comment (Tirokk) [inline webapp/locales/de.json:639]: > [roschaefer](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2020-02-19T22:31:43Z" title="Wednesday, February 19th 2020, 11:31:43 pm +01:00">Feb 19, 2020</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -635,8 +635,8 @@
       "success": "Deine Daten wurden erfolgreich aktualisiert!"
     },
     "deleteUserAccount": {
-      "accountDescription": "Sei dir bewusst, dass deine Beiträge und Kommentare für unsere Community wichtig sind. Wenn du sie trotzdem löschen möchtest, musst du sie unten markieren.",
-      "accountWarning": "Dein Konto, deine Beiträge oder Kommentare kannst du nach dem Löschen <b>WEDER VERWALTEN NOCH WIEDERHERSTELLEN!</b>",
+      "accountDescription": "Sei Dir bewusst, dass Deine Beiträge und Kommentare für unsere Community wichtig sind. Wenn Du sie trotzdem löschen möchtest, musst Du sie unten markieren.",
+      "accountWarning": "Dein Konto, Deine Beiträge oder Kommentare kannst Du nach dem Löschen <b>WEDER VERWALTEN NOCH WIEDERHERSTELLEN!</b>",
```


Both caps and bold, maybe a just bold would be enough?

**Review by Mogge — APPROVED** [review]
Looks very good to me

**Review by Mogge — APPROVED** [review]
translate

@Mogge the translations are so much easier to compare now!

**Pull request closed without merging**